### PR TITLE
Added an array with 3 objects, use the map function to iterate over i…

### PR DESCRIPTION
**Pull request by Karla1981**
…t and return list item

### CODE REVIEW

**Comment by Karla1981** [comment]
Lesson-1-1

**Comment by obrienwritescode** [comment]
Hello @Karla1981, this PR is within the Code-the-Dream-School Github organization.  Could you please fork the ctd-react-albatross repository and then create PRs within it.  Instructions can be found here: https://github.com/Code-the-Dream-School/ctd-react-albatross/wiki/General-Instructions#project-setup

File: src/App.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+];
+
+function App() {
+  return (
+    <div>
+      <header>
+      <h2>ToDo List :</h2> 
+        <ul>
+          {todoList.map( function(listItem) {
+            return <li>{listItem.id} {listItem.title}</li>;
```
Comment (obrienwritescode) [inline src/App.js:23]: You `<li>` is missing one required attribute that should be present.  Take a closer look at the assignment and see if you can find that attribute.

**Review by obrienwritescode — CHANGES_REQUESTED** [review]
- Fork the CTD repository into your own and then always merge into your own repository. 
- Verify the missing attribute for `<li>` elements.

**Pull request closed without merging**